src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is now working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck has stopped working");
```

Comment:
Similarly, the message 'Truck has stopped working' should be more informative. Include the class name in the message to clearly indicate which machine has stopped working.